test2code-agent/src/main/kotlin/com/epam/drill/plugins/test2code/Ast.kt

```diff
@@ -45,13 +46,19 @@ open class ProbeCounter : ClassProbesVisitor() {
 }
 
 class ClassProbeCounter(val name: String) : ProbeCounter() {
-    val astClass = newAstClass(name, ArrayList())
+    val astClass = newAstClass(name, ArrayList(), ArrayList())
 
     override fun visitMethod(
         access: Int, name: String?, desc: String?, signature: String?, exceptions: Array<out String>?
     ): MethodProbesVisitor {
         return MethodProbeCounter(astClass.methods as MutableList)
     }
+
+    override fun visitAnnotation(desc: String?, visible: Boolean): AnnotationVisitor? {
+        val annotationName = Type.getType(desc).className
+        (astClass.annotations as MutableList).add(annotationName)
```

Comment:
Instead of using ` as MutableList` I would change the type of this field to `MutableList`. e.g, `annonations = MutableList<String>()`

Reply:
done.